core/src/main/scala/cats/instances/invariant.scala

```diff
@@ -0,0 +1,168 @@
+package cats.instances
+
+import cats.kernel._
+import cats.InvariantMonoidal
+
+trait InvariantInstances {
```

Comment:
looks like these are all `InvariantMonoidal` not just `Invariant`. Should we put them in a different trait name?